src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -130,14 +137,12 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example to allow only one and only parameterized annotation on the same line
+ * Example to allow one single parameterless annotation on the same line is same as
+ * the example provided for default configuration.
```

Comment:
it make sense to make single example code block and reuse it in all examples to show how behavior is different when properties change their values. please do.

Reply:
Done